src/components/Form.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from "react";
+import styles from "./Form.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { TODOLIST } from "../assests/Constants";
+
+const ToDoForm = ({ inputText, setInputText, list }: any) => {
+  const inputTextHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputText(e.target.value);
+  };
+
+  const addToDoHandler = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const res = await fetch(TODOLIST, {
+        method: "POST",
+        headers: {
+          Accept: "application/json",
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          title: "food",
```

Comment:
Edit: this is why the casing is different and the id is a decimal. Maybe `Math.floor(Math.random() * 1000)`.